include/macros.h

```diff
@@ -45,6 +45,9 @@
                       (panel)->numCols * (panel)->row + \
                       (panel)->col]
 
+#define IS_ID_BADGE(itemID) (itemID >= 0xE0 && itemID <= 0x154)
+#define IS_ID_ITEM(itemID) (itemID >= 0x10 && itemID <= 0xDA)
```

Comment:
Potentially wise to use the enum values here: (also looks like they're fairly arbitrary? 'ITEM_JELLY_SHROOM1' is an 'item' but 'ITEM_JELLY_SHROOM2' isn't?)
```suggestion
#define IS_ID_BADGE(itemID) (itemID >= ITEM_SPIN_SMASH && itemID <= ITEM_ATTACK_FX_F5)
#define IS_ID_ITEM(itemID) (itemID >= ITEM_KOOPA_FORTRESS_KEY && itemID <= ITEM_JELLY_SHROOM1)
```
There's also already a macro in `enums.h` next to `enum ItemIds` ([here](https://github.com/pmret/papermario/blob/c2fac0235d61215df13b463e4e5267fadad73d5e/include/enums.h#L910)) so it might be wise to put these two next to it, since they're directly related